packages/execution-environments/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@metamask/execution-environments",
+  "version": "0.1.0",
```

non-blocking: If we start the package at version `0.0.0`, it tends to work better with our release automation, although it probably doesn't matter for monorepos as long as the version is less than or equal to the highest version in the monorepo.

```suggestion
  "version": "0.0.0",
```